dashboard/internal/data/stats.go

```
@@ -0,0 +1,443 @@
+		case "paris":
+			return "Paris"
+		case "amsterdam":
+			return "Amsterdam"
+		case "zurich", "zürich":
+			return "Zurich"
+		case "dublin":
+			return "Dublin"
+		case "vienna", "wien":
+			return "Vienna"
+		default:
+			return strings.Title(c)
```

golangci-lint (2.12.2)

[error] SA1019: strings.Title has been deprecated since Go 1.18 and an alternative has been available since Go 1.0: The rule Title uses for word boundaries does not handle Unicode punctuation properly. Use golang.org/x/text/cases instead. 

(staticcheck)